content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -54,6 +54,14 @@ stateful tasks.
 
 {{< figure src="/images/docs/scheduling-framework-extensions.png" title="scheduling framework extension points" class="diagram-large">}}
 
+### PreEnqueue {#pre-enqueue}
+
+These plugins are called prior to adding Pods to the internal active queue, where Pods are marked as
+ready for scheduling.
+
+Only when all PreEnqueue plugins return `Success`, the Pod can enter the aforementioned scheduling
+cycle. Otherwise, it's moved and parked in the internal unschedulable Pods pool.
```

Comment:
```suggestion
Only when all PreEnqueue plugins return `Success`, the Pod is allowed to into the active queue. Otherwise, it is placed in the internal unschedulable Pods list.
```

Reply:
Also add: Pods that are in the unschedulable Pods list do not get an `Unschedulable` condition.

Reply:
Updated.